models/song.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+const song = new mongoose.Schema({
```

Comment:
I would consider using `Song` rather than `song` since models and classes can be non camelCase :)

Reply:
Working on it!